torchci/components/JobConclusion.module.css

```diff
@@ -51,46 +51,46 @@
 }
 
 .skipped {
-  color: grey;
+  color: #9e9e9e; /* Grey that works in both light and dark mode */
```

Comment:
I don't think we need a comment every time we change name, but would be nice to have color names/constants in one place and not repeating it every time (also I don't need a comment in the code that this looks nice in both modes)